tests/unittest.py

```diff
@@ -87,6 +87,7 @@ def setUp(self):
         self.reset_peer_id_pool()
         self.rng = Random()
         self._pending_cleanups = []
+        self.pubsub = PubSubManager(self.clock)
```

Comment:
Why are you creating one global pubsub shared among all peers? In the mainnet, each peer has its own pubsub and they do not share events. I have a strong feeling that this change is a mistake.

Reply:
That had been done because to call `create_peer()` I have to pass an `EventManager`(due to https://github.com/HathorNetwork/hathor-core/pull/522#discussion_r1098762460), and to instantiate one I needed a `PubSub` instance. So I moved it to `self` to have access to the instance before crating the `EventManager`.

I understand the problem now and have changed the code so `create_peer()` optionally receives a `PubSub` instance (which I have to pass in my tests), or creates a new one (as was being done before).

Reply:
Ok. Thanks!